spec/passfort/http_spec.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Passfort::Http do
+  shared_examples "an error response" do
+    before do
+      stub_request(method, api_path).
+        to_return(status: status, body: body)
+    end
+
+    # rubocop:disable RSpec/MultipleExpectations
+    it "raises a specific APIError" do
+      is_expected.to raise_error do |error|
```

Comment:
Oh actually you can do with without multiple expectations:
```ruby
is_expected.to raise_error(instance_of(error_class))
```

Reply:
👍